client/store/cart.js

```diff
@@ -31,7 +33,10 @@ export const getCart = (userId) => {
   // take in userId, return cart that belongTo that user
   return async (dispatch) => {
     try {
-      const axiosResponse = await axios.get(`/api/users/${userId}/cart`);
+      const token = window.localStorage.getItem(TOKEN);
```

Comment:
NEW: get token from localStorage and pass to API route as header before allowing cart access